tutorials/io/30_reading_fnirs_data.py

```diff
@@ -25,7 +25,6 @@
              Manual modification of channel names and metadata
              is not recommended.
 
```

Comment:
```suggestion

.. note:: To provide a consistent interface across different measurement devices and
          file types, MNE-Python uses a standard ordering of channels. MNE-Python
          internally orders channels by ascending source number. When there are multiple 
          channels with the same source number, they are ordered by ascending detector number.
          The ordering of channels is done automatically when data is imported.
          Therefore, the ordering of channels within MNE-Python may be different to what
          was provided by the hardware vendor.
```

Reply:
We can add this but hopefully it's temporary and will be removed before 1.1. Sound good?

Reply:
Allowing arbitrary ordering of channels and not requiring alternating ordering of wavelengths would absolutely be the best solution. This would also make it easier for systems with more than two wavelengths of light etc. Sounds great @larsoner